docker-storage-setup.1

```diff
@@ -172,30 +168,28 @@ The options below should be specified as values acceptable to 'lvextend -L':
 
 ROOT_SIZE: The size to which the root filesystem should be grown.
 
-DATA_SIZE: The desired size for the docker data LV.  Defaults to using
-           40% free space in the VG after the root LV and docker
-           metadata LV have been allocated/grown.
-
-           DATA_SIZE can take values acceptable to "lvcreate -L" as
-           well as some values acceptable to "lvcreate -l". If user
-           intends to pass values acceptable to "lvcreate -l", then
-           only those values which contains "%" in syntax are
-           acceptable.  If value does not contain "%" it is assumed
-           value is suitable for "lvcreate -L".
-
-MIN_DATA_SIZE:
-           Specifies the minimum size of the data volume. If sufficient
-           free space is not available, the pool creation will fail.
-
-           Value should be a number followed by a optional suffix.
-           "bBsSkKmMgGtTpPeE" are valid suffixes. If no suffix is
-           specified then value will be considered as megabyte unit.
-
-           Both upper and lower case suffix represent same unit of
-           size. Use suffix B for Bytes, S for sectors as 512 bytes, K
-           for kibibytes (1024 bytes), M for mebibytes
-           (1024 kibibytes), G for gibibytes, T for tebibytes, P for
-           pebibytes and E for exbibytes.
+DATA_SIZE: The desired size for the container runtime data LV.
+	Defaults: 40% free space in the VG after the root LV and container
```

Comment:
I think we would need to go over this together.  There are a ton of words in this man page and default config already,  not sure you will ever be able to fully express what you mean in a manner people would understand.